components/fields/text/TextField.tsx

```diff
@@ -48,11 +53,12 @@ export function TextField (props: TextFieldProps) {
         props?.change,
         props?.changeState,
         props?.value,
-        props?.model?.required ?? false,
-        props?.model?.minLength,
-        props?.model?.maxLength
-    );
+         props?.model?.required ?? isRequired,
+        props?.model?.minLength ? props?.model?.minLength : props?.minLength,
```

Comment:
Change order, since more specific direct props value should override values from model.

Eg. `props?.minLength !== undefined ? props?.minLength : props?.model?.minLength`

It's important to check also `undefined`, since `0` is also `false`.

Same for maxLength